cli/io/fio.py

```diff
@@ -0,0 +1,114 @@
+import itertools
+
+from cli import Cli
+from cli.utilities.packages import Package
+
+
+class FioError(Exception):
+    pass
+
+
+class Fio(Cli):
+    def __init__(self, client):
+        super(Fio, self).__init__(client)
+        self.client = client
+
+    def install_fio(self):
```

Comment:
This should be part of installation step. Here in library assume that this package is present.